npm.js

```diff
@@ -66,6 +73,8 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				pkg = transform(loader, load, pkg);
```

Comment:
These packages don't have a load object, they are not imported by the loader, only fetched in npm-crawl. This includes the one in npm.js, btw, that should not be using `load` there. Instead I'd create a fake load object to pass in that looks like:

```js
{
			address: pkg.fileUrl,
			name: pkg.fileUrl,
			metadata: {}
		}
```